com.github.alainm23.byte.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "com.github.alainm23.byte",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.alainm23.byte",
+    "finish-args": [
+        /* X11 + XShm */
+        "--share=ipc", "--socket=x11",
+        /* Wayland */
+        "--socket=wayland",
+        /* Filesystem */
+        "--filesystem=host",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        /* Network (Radio + Covers) */
+        "--share=network",
+        /* dconf */
+        "--metadata=X-DConf=migrate-path=/com/github/alainm23/byte/",
+        /* Audio */
+        "--socket=pulseaudio",
+        /* Notifications */
+        "--talk-name=org.freedesktop.Notifications",
+        /* Inhibitor */
+        "--talk-name=org.freedesktop.ScreenSaver",
```

Comment:
`Gio.Application` probably exposes what you use for this and you can remove this hole too.